From c5c8081123de80a2a322633a4956a6887e1f9a87 Mon Sep 17 00:00:00 2001
From: Carlo Caione <ccaione@baylibre.com>
Date: Wed, 28 Nov 2018 16:40:57 +0000
Subject: [PATCH 45/55] recovery: apollo: No initramfs on recovery partition

Signed-off-by: Carlo Caione <ccaione@baylibre.com>
---
 include/env_mender.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/env_mender.h b/include/env_mender.h
index c10dc792ae..e823f6dffb 100644
--- a/include/env_mender.h
+++ b/include/env_mender.h
@@ -79,6 +79,7 @@
     "else " \
         "setenv mender_boot_part_name " APOLLO_ROOTFS_PART_R_NAME "; " \
         "setenv mender_boot_part_hex " __stringify(APOLLO_ROOTFS_PART_R_NUMBER_HEX) "; " \
+        "setenv ramdisk_addr_r \"\";" \
     "fi; fi; " \
     "setenv mender_kernel_root_name ${mender_boot_part_name}; " \
     "setenv mender_uboot_root " MENDER_UBOOT_STORAGE_INTERFACE " " __stringify(MENDER_UBOOT_STORAGE_DEVICE) ":${mender_boot_part_hex}; " \
-- 
2.19.1

